src/utils/searchViewHelper.ts

```diff
@@ -12,79 +13,65 @@ import {
   fetchRevisionByID,
   fetchRevisionsByAuthor,
 } from '../thunks/searchThunk';
+import type { Repository } from '../types/state';
 
-let timeout = null;
+let timeout: NodeJS.Timeout;
```

Comment:
I'm a bit surprised with the `NodeJS` bit -- and at the same time I'm happy that it's not simply a `number` so that its use is better framed. Let's keep it this way for now but we may need to adjust in the future.